views/dashboard/index.html.twig

```diff
@@ -211,6 +211,9 @@
                         renderTotalProgressPlot('spinner');
                         PLOTS_SHOWN['total-progress-nav'] = true;
                     }
+                    // Send pageview event for tab
+                    ga('set', 'page', '/dashboard/total-progress');
+                    ga('send', 'pageview');
```

Comment:
I think you can just pass the page as the third parameter

```js
ga('send', 'pageview', '/dashboard/total-progress');
```

See https://developers.google.com/analytics/devguides/collection/analyticsjs/pages#implementation

Reply:
Replaced. I could see where if I were sending additional events, the `set` command might be handy.